### PR TITLE
(#850) Added music to the glowing tile

### PR DESCRIPTION
**Pull request by AppleC4T**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot to Indidate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-851-fossasia-labyrinth.surge.sh or at https://applec4t.github.io/labyrinth/

- Fixes https://github.com/fossasia/labyrinth/issues/850

### The problem I want to solve or the facility I want to improve is/are

Add glitched/dark music to the dark glowing tile
### My steps to solve it

Added music to the tile
### Screenshots, Location of Changes

image

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -314,6 +314,13 @@ const door = {
       this.wallRight = this.createImage("tiles/rooms/wall/dark.svg");
       this.ground = this.createImage("tiles/animations/glow.svg"); 
     },
+		 visit: function() {
```
Comment (abishekvashok) [inline js/tiles.js]: Properly intend it pls

Comment (AppleC4T) [reply js/tiles.js]: @abishekvashok check now, should be ok and if not - let me know

Comment (YashKumarVerma) [reply js/tiles.js]: Resolved.

**Comment by AppleC4T** [comment]
Thanks.

**Pull request merged**